subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/ValidationActions.java

```diff
@@ -35,7 +38,11 @@ public void doValidate(String propertyName, Object value, TaskValidationContext
         public void doValidate(String propertyName, Object value, TaskValidationContext context) {
             File file = toFile(context, value);
             if (!file.exists()) {
-                context.visitError(String.format("File '%s' specified for property '%s' does not exist.", file, propertyName));
+                if (Files.isSymbolicLink(file.toPath())) {
+                    LOGGER.warn("File '{}' specified for property '{}' is a broken symbolic link.", file, propertyName);
```

Comment:
should the validation ignore the broken links completely, or is it okay to warn?